src/components/button/button.component.js

```diff
@@ -41,97 +96,46 @@ const renderStyledButton = (buttonProps) => {
 
   return (
     <StyledButton
-      as={!disabled && href ? "a" : "button"} // legacy link button feature
+      as={!disabled && href ? "a" : "button"}
```

Comment:
This means that disabled links would render as a button. Should we be handling this on the `a` element instead by setting the cursor and blocking the click event?

Not blocking for this PR.

Reply:
Good point I'll change that